frontend/components/AddHostsModal/PlatformWrapper/PlatformWrapper.tsx

```diff
@@ -301,7 +301,7 @@ const PlatformWrapper = ({
               information below.
             </p>
             <InfoBanner className={`${baseClass}__chromeos--instructions`}>
-              For a step-by-step guide, see the documentation page for{" "}
+              For a step-by-step guide, see the documentation page for&nbsp;
```

Comment:
I noticed in my `ui-reskin` the {" "} wasn't rendering with whatever changes I did to info banner so merging this in worked, sorry got strung in here